src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+if ('NDEFWriter' in window) { /* Write NFC tags */ }
+```
+
+### Terminology {: #terminology }
+
+NFC stands for Near Field Communications, a short-range wireless technology
+operating at 13.56 MHz which enables communication between devices at a distance
+less than 10 cm and a transmission rate of up to 424 kbit/s.
+
+An NFC tag is a passive NFC device, powered by magnetic induction when an active
+NFC device (.e.g a phone) is in proximity range.
+
+<figure class="w-figure">
+  <img src="./nfc-tag.jpg" alt="Photo of a transparent NFC tag">
+  <figcaption class="w-figcaption">A transparent NFC tag</figcaption>
+</figure>
+
+The `NDEFReader` and `NDEFWriter` objects are the two entry points in Web NFC
+that expose functionality for preparing reading and/or writing actions that are
+fulfilled when an NDEF tag comes in proximity range. The `NDEF` in `NDEFReader`
+and `NDEFWriter` stands for NFC Data Exchange Format, a lightweight binary
+message format standardized by the [NFC Forum].
+
+The `NDEFReader` object is for acting on incoming NDEF messages from NFC tags
+while the `NDEFWriter` object is for writing NDEF messages to NFC tags within
+range.
+
+An NFC tag that supports NDEF is like a post-it note. Anyone can read it, and
+unless it is read-only, anyone can write to it. It contains a single NDEF
+message which encapsulates one or more NDEF records. Each NDEF record is a
+binary structure that contains a data payload, and associated type information.
+Web NFC supports the following NFC Forum standardized record types: empty, text,
+URL, smart poster, MIME type, absolute URL, external type, unknown, and local
+type.
+
+<figure class="w-figure">
+  <img src="./ndef-message-diagram.png" alt="Diagram of an NDEF message">
+  <figcaption class="w-figcaption">Diagram of an NDEF message</figcaption>
+</figure>
+
+### Scan NFC tags {: #scan }
+
+To scan NFC tags, first instantiate a new `NDEFReader` object. Calling `scan()`
+returns a promise. The [user may be prompted] if access was not previously
+granted. The promise will resolve if the following conditions are all met:
+
+- User has allowed website to interact with NFC devices when they tap their
+  phone.
+- User's phone supports NFC.
+- User has enabled NFC on their phone.
+
+Once the promise is resolved, incoming NDEF messages are available by
+subscribing to "reading" events via an event listener. You should also subscribe
+to "error" events to be notified when incompatible NFC tags are in proximity
+range.
+
+```js
+const reader = new NDEFReader();
+reader.scan().then(() => {
+  console.log("Scan started successfully.");
+  reader.onerror = () => {
+    console.log("Argh! Cannot read data from the NFC tag. Try another one?");
+  };
+  reader.onreading = event => {
+    console.log("NDEF message read.");
+  };
+}).catch(error => {
+  console.log(`Error! Scan failed to start: ${error}.`);
+});
+```
+
+When an NFC tag is in proximity range, a NDEFReadingEvent event is fired. It
```

Comment:
within?

Reply:
I'm not sure. @jpmedley will know.

Reply:
'in' is fine. 'Proximity' is redundant. Say 'in range' or 'in proximity'.